modules/core/src/bitcoin.ts

```diff
@@ -12,14 +12,14 @@ export function getNetwork(network?: V1Network): utxolib.Network {
   return utxolib.networks[network];
 }
 
-export function makeRandomKey(): utxolib.ECPair {
-  return utxolib.ECPair.makeRandom({ network: getNetwork() });
+export function makeRandomKey(): utxolib.ECPair.ECPairInterface {
+  return utxolib.ECPair.makeRandom({ network: getNetwork() as any });
```

Comment:
These are sadly necessary since the `bech32` property is required in `bitcoinjslib.Network` but is optional in `utxolib.Network`.

It is safe in almost all instances because the `bech32` property is only actually relevant in a small minority of cases.